tsconfig.json

```diff
@@ -5,27 +5,20 @@
     "jsx": "react",
     "lib": [
       "dom",
-      "es5",
-      "scripthost",
-      "es2015",
-      "es2015.promise",
-      "es2016.array.include",
-      "es2018",
-      "es2020",
-      "es2021",
-      "es2022"
+      "dom.iterable",
+      "esnext"
     ],
     "module": "esnext",
     "moduleResolution": "node",
     "noEmit": true,
     "paths": {
       "@openmrs/*": ["./node_modules/@openmrs/*"],
       "__mocks__": ["./__mocks__"],
-      "tools": ["./tools"],
+      "tools": ["./tools"]
     },
     "resolveJsonModule": true,
     "skipLibCheck": true,
     "strictNullChecks": true,
-    "target": "esnext",
+    "target": "esnext"
```

Comment:
Should we change the target to be more conservative (maybe es2022?), as older browsers do not support bleeding edge features?

Reply:
The ts-config is actually pretty irrelevant. We just use ts for type-checking. [This](https://github.com/openmrs/browserslist-config-openmrs/blob/master/src/browserslist-config-openmrs.js) is what the bundler uses for the actual output.